pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -559,6 +559,28 @@ func (r *Builder) Tasks(vmRef ref.Ref) (list []*plan.Task, err error) {
 	return
 }
 
+func (r *Builder) PreferenceName(vmRef ref.Ref, configMap *core.ConfigMap) (name string, err error) {
+	if configMap == nil {
+		err = liberr.New("can't map the OS, missing config map")
+		return
+	}
+	vm := &model.Workload{}
+	if err = r.Source.Inventory.Find(vm, vmRef); err != nil {
+		err = liberr.Wrap(
+			err,
+			"VM lookup failed.",
+			"vm",
+			vmRef.String())
+		return
+	}
+	name, ok := configMap.Data[vm.OSType]
+	if !ok {
+		err = liberr.Wrap(err,
+			"nothing fits the input OS")
```

Comment:
did you check that this log looks alright? the signature of liberr.Wrap is `(err error, kvpair ...interface{})`

Reply:
added more data to align with the rest of the liberr wrap

Reply:
ok, [filed an issue to fix it separately](https://github.com/kubev2v/forklift/issues/741)